### PR TITLE
Add quick start section

### PR DESCRIPTION
**Pull request by dmjio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -116,6 +134,220 @@ viewModel x = div_ [] [
...
+extra-deps: []
+
+setup-info:
+  ghcjs:
+    source:
+      ghcjs-0.2.0.9006020_ghc-7.10.3:
+         url: http://ghcjs.tolysz.org/lts-6.20-9006020.tar.gz
+         sha1: a6cea90cd8121eee3afb201183c6e9bd6bacd94a
+
+- location:
```
Comment (cocreature) [inline README.md]: Is there a reason to choose this specific commit? If not I would suggest to just add an entry in the `extra-deps` section that fetches `miso-0.4.0.0` from hackage.

Comment (dmjio) [inline README.md]: @cocreature I was wondering the same, but wasn't sure how to specify to fetch from `hackage`.

Comment (dmjio) [inline README.md]: 🤦‍♂️ , can just add `miso-0.4.0.0` to `extra-deps`

**Comment by dmjio** [comment]
@cocreature 🍻

**Pull request merged**